CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.54.3] - 2024-07-17
+### Added
+- [Feature Preview] Support for `enum` as container property type in the data modeling APIs. Note that this is not
```

Comment:
Minor bump as it is an experimental feature.

Reply:
Minor?